addons/contour/1.0.1/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+resources:
```

Comment:
You can create a [template](https://github.com/replicatedhq/kurl/blob/master/addons/velero/1.2.0/tmpl-namespace.yaml) that references a bash variable and then [render it](https://github.com/replicatedhq/kurl/blob/master/addons/velero/1.2.0/install.sh#L24). It's also a good idea to [pre-create it](https://github.com/replicatedhq/kurl/blob/master/addons/velero/1.2.0/install.sh#L26) to avoid race conditions.

You can add a function named `contour_pre_init` to [set the namespace](https://github.com/replicatedhq/kurl/blob/master/addons/velero/1.2.0/install.sh#L2) if unset.

Reply:
@areed I removed the namespace from all the yaml docs, but just a quick question, in the `rbac.yaml` I removed the `namespace` from `subjects:` also, not just `metadata:`, is that okay? Or should I leave it like below or would the override in `kustomization.yaml` takes care of this?

```
subjects:
  - kind: ServiceAccount
    name: contour
    namespace: projectcontour
```